intro_to_r/intro_to_r.Rmd

```diff
@@ -35,21 +35,21 @@ and interpreting the output. These commands and their syntax have evolved over
 decades (literally) and now provide what many users feel is a fairly natural way
 to access data and organize, describe, and invoke statistical computations.
 
-To get you started, enter the following commands at the R prompt (i.e. right after
-`>` on the console).  You can either type it in manually or copy and paste it
+To get you started, enter all commands at the R prompt (i.e. right after
+`>` on the console); you can either type them in manually or copy and paste them
 from this document.
 
-R has a number of additional packages that help you perform specific tasks. In order to use these packages, they must be installed (you only have to do this once) and loaded (you have to do this every time you start an R session). 
+R has a number of additional packages that help you perform specific tasks. In order to use these packages, they must be installed (you only have to do this once) and loaded (you have to do this every time you start a new R session). 
 
 ```{r opts, eval=TRUE, echo=FALSE}
 library(knitr)
 opts_chunk$set(eval=FALSE)
 ```
 
-If you have not installed the packages already, run these commands:
+If you have not installed the packages already, run the commands below. It may take a few minutes; you'll know when the packages have finished installing when you see the R prompt return in your console.
 ```{r load-devtools-oilabs, message = FALSE, eval=FALSE}
```

Comment:
The block originally assumed `devtools` package was installed for all users on RStudio server. I'm guessing however this issue will be obviated once the `oilabs` package is available on CRAN. 

Also, when my students tried to install `dplyr` on RStudio Server, it required installation of `Rcpp` separately. I unsuccessfully tried to replicate the error on my local RStudio, so I left this edit out. It could be a configuration of RStudio Server issue.